modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/Oauth2HashAlgorithmTestCase.java

```diff
@@ -247,9 +246,9 @@ public void testValidateAccessToken() throws Exception {
 
     private void registerAndDeployApplication() throws Exception {
```

Comment:
This method is not deploying any application, right? Shall we rename it?

Reply:
Fixed with https://github.com/wso2/product-is/pull/16271/commits/e9343598362e5021e1f6832b61ef1997495afeda
Changed the method name to `createApp`